### ISSUE TITLE
False-positive "Argument never used" when using backslash in raw string in format macro

### ISSUE THREAD

**Issue by dima74** [intellij-rust/intellij-rust#5385]
## Environment

* **IntelliJ Rust plugin version:** 0.2.123.3094-201-nightly
* **Rust toolchain version:** 1.43.0 (4fb7144ed 2020-04-20) x86_64-unknown-linux-gnu
* **IDE name and version:** CLion 2020.1.1 (CL-201.7223.86)
* **Operating system:** Linux 5.6.11-arch1-1

## Problem description
![image](https://user-images.githubusercontent.com/6505554/82115130-5b318e00-977a-11ea-89ac-49c2c9734e56.png)

## Steps to reproduce
```rust
fn main() {
    println!(r"\{}", 1);
}
```
